<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#frame, #frame_3 { background-color: white}
#line, #line_2 {border: 3px solid #b0b0b0; margin:0; padding:0}
QPushButton{
background-color:#ffdf00;
border-radius: 30;
}
QPushButton:hover{ background-color: #f1d100}
QPushButton:pressed{background-color:#dfc200 }
QPushButton:disabled{background-color:#cbcbcb }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Equinox Bold</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Alarme Novo!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>2</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>2</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>25</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>25</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_138">
        <property name="styleSheet">
         <string notr="true">#frame_138{
background-color: #989898; border-radius: 50;
padding:0; margin:0; border: 2px solid #888888
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_21">
         <property name="spacing">
          <number>35</number>
         </property>
         <property name="leftMargin">
          <number>35</number>
         </property>
         <property name="topMargin">
          <number>35</number>
         </property>
         <property name="rightMargin">
          <number>35</number>
         </property>
         <property name="bottomMargin">
          <number>35</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_139">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">#frame_139{
background-color: lightgray; padding:20; margin:0; border:0;
border-radius: 15;
border: 5px solid #888888
}

QFrame{
padding:0; margin:0; border: 4px solid #ffdf00
}
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_22">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_141">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_23">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QTableWidget" name="alarm_list_widget">
                 <property name="font">
                  <font>
                   <family>Equinox Bold</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"> * { border:0; margin: 0; padding:0}

QHeaderView::section {
    padding: 5 20;
    border: 1px solid #ffdf00;
	border-bottom: 2px solid #ffdf00;
	border-top: 0;
	background-color: white
}

QHeaderView::up-arrow, QHeaderView::down-arrow {
	color: black;
	width: 25; height: 25;
	subcontrol-position: left;
	
}

QTableWidget {
	gridline-color: #ffdf00;
}

QTableWidget::item {
	border: 1px solid #ffdf00;
	border-right: 0; border-left: 0;
	padding: 0 20;
}

QTableWidget::item:selected{
	background-color: #ababab
}</string>
                 </property>
                 <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                 </property>
                 <property name="alternatingRowColors">
                  <bool>false</bool>
                 </property>
                 <property name="selectionMode">
                  <enum>QAbstractItemView::SingleSelection</enum>
                 </property>
                 <property name="selectionBehavior">
                  <enum>QAbstractItemView::SelectRows</enum>
                 </property>
                 <property name="sortingEnabled">
                  <bool>true</bool>
                 </property>
                 <property name="rowCount">
                  <number>0</number>
                 </property>
                 <property name="columnCount">
                  <number>2</number>
                 </property>
                 <attribute name="horizontalHeaderCascadingSectionResizes">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="horizontalHeaderDefaultSectionSize">
                  <number>300</number>
                 </attribute>
                 <attribute name="horizontalHeaderHighlightSections">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="horizontalHeaderStretchLastSection">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="verticalHeaderVisible">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="verticalHeaderHighlightSections">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="verticalHeaderShowSortIndicator" stdset="0">
                  <bool>true</bool>
                 </attribute>
                 <column>
                  <property name="text">
                   <string>TIME MESSAGE</string>
                  </property>
                  <property name="font">
                   <font>
                    <family>Equinox Bold</family>
                    <pointsize>16</pointsize>
                   </font>
                  </property>
                  <property name="textAlignment">
                   <set>AlignLeading|AlignVCenter</set>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>MESSAGE</string>
                  </property>
                  <property name="font">
                   <font>
                    <family>Equinox Bold</family>
                    <pointsize>16</pointsize>
                   </font>
                  </property>
                  <property name="textAlignment">
                   <set>AlignLeading|AlignVCenter</set>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>2</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>2</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>224</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_go_alarm_screen">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>65</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>65</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Equinox Bold</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Ir para tela
de alarmes</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>225</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_cancel">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>65</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>65</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Equinox Bold</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">* {
background-color: #eece00
}

QPushButton:hover{
background-color: #ddbd00
}

QPushButton:pressed {
background-color: #ccac00
}
QPushButton::disabled{background-color:#cbcbcb}</string>
        </property>
        <property name="text">
         <string>Fechar</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>224</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
